src/tokens/core.js

```diff
@@ -91,14 +125,35 @@ export default {
     SuperJumbo: 96,
   },
   Time: {
-    Short: "300ms",
-    Med: "500ms",
-    Long: "1000ms"
+    Short: 300,
+    Med: 500,
+    Long: 1000,
   },
-  Easing: {
-    Default: "cubic-bezier(.475,.425,0,.995)"
+  Easing: target => {
+    if (target === targets.REACT) {
+      return {
+        Default: "cubic-bezier(.475,.425,0,.995)"
+      }
+    }
+
+    // TODO: Define type cubic-bezier?
+    return {
+      Default: {
+        x1: 0.475,
+        y1: 0.425,
+        x2: 0,
+        y2: 0.995
+      }
+    }
   },
+  // Convenience values as web consumes both constantly
   MediaQuery: {
```

Comment:
This should work fine. I imagine this also gives us more flexibility to have different breakpoints for web vs phones if need be.